pakiti-client

```diff
@@ -369,9 +368,42 @@ sub encrypt_report ($) {
 # send a formatted report
 #
 
+sub https {
+    my ($url, $report) = @_;
+    my($ua, $response);
+    my($code,$body);
+    eval { require LWP::UserAgent; LWP::UserAgent->import; };
+    if ($@) {
+	if ($Option{"disable-tls-checks"}) {
+            ($code,$body) = `curl -sik -X POST $url -d "$report"` =~m{\A\S+ (\d+) .*?\r?\n\r?\n(.*)}s;
+        } else {
+            ($code,$body) = `curl -si -X POST $url -d "$report"` =~m{\A\S+ (\d+) .*?\r?\n\r?\n(.*)}s;
+        }
+        if ($body =~ m/200 OK/) {
```

Comment:
this check seems potentially fragile. It'd be good to check the exec code of curl. For the check of HTTP code it'd be better to make sure you really check the correct HTTP header not just a random "200 OK" string in the message.